editor/resources/templates/template.cp

```diff
@@ -4,7 +4,10 @@ layout (local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
 
 layout(rgba32f) uniform image2D texture_out;
 
-uniform vec4 color;
+uniform uniforms
+{
+    vec4 color;
+};
```

Comment:
This shader is using the new pipeline, which doesn't allow non-opaque uniforms to be outside a block.